### DISCUSSION TITLE
With engine 19.03 docker incorporated minikube

### DISCUSSION THREAD

**Discussion by spyd3rw3b** [BretFisher/ama#196] [Q&A]
with docker desktop 2.1 and engine 19.03 minikube was integrated to the menu to start kubernetes (see attached)...what is the benefit for this compared to auto starting it from something like portainer?

![docker_dektop](https://user-images.githubusercontent.com/6453472/70768062-33508100-1d29-11ea-9397-51360917a111.png)

**Comment by BretFisher** [comment]
That's not an integration, that's just the kubectl context list you're seeing, and a quick way to change it. It's a GUI version of the cli `kubectl config set-context`. 

By default docker sets it to `docker-desktop`, but if you installed minikube, which adds its own context to `~/.kube/config`, you can change it quickly in the menu GUI 👍
